playgrounds/vite/package.json

```diff
@@ -8,6 +8,7 @@
     "preview": "vite preview",
     "clean": "rimraf .turbo && rimraf node_modules && rimraf dist"
   },
+  "type": "module",
```

Comment:
nit: could you move this near the top of `package.json` so it's clear immediately that this project is ESM?

Reply:
https://github.com/iTwin/iTwinUI/pull/1549/commits/06470baede198e4d18a9d0dd2c1a29e79f07644d: Reordered the keys of a few `package.json`s to bring `type` near the top. Also, did this uniformity with other `package.json` files in the monorepo.